Python/rerf/rerfClassifier.py

```diff
@@ -208,10 +208,16 @@ def fit(self, X, y):
         self.forest_ = pyfp.fpForest()
 
         if self.projection_matrix == "Base":
-            forestType = "binnedBase"
+            if self.oob_score:
+                forestType = "rfBase"
+            else:
+                forestType = "binnedBase"
             self.method_to_use_ = None
         elif self.projection_matrix == "RerF":
-            forestType = "binnedBaseTern"
+            if self.oob_score:
+                forestType = "rerf"
+            else:
+                forestType = "binnedBaseTern"
             self.method_to_use_ = 1
         elif self.projection_matrix == "S-RerF":
             forestType = "binnedBaseTern"  # this should change
```

Comment:
Does s-rerf support oob?  If so, we need to do the same switch statement for forest type. If not, we should throw a warning if oob chosen

Reply:
No, it does not support job because it uses the binnedBaseTern forestType.